index.html

```diff
@@ -0,0 +1,158 @@
+           <li><a href="#" class="link-nav">Program</a></li>
+           <li><a href="#" class="link-nav">Sponsor</a></li>
+           <li><a href="#" class="link-nav">Join</a></li>
+        </ul>
+      </div>
+
+       <!-- end of mobile menu dynamic -->
+
+    <div class="desktop-menus">
+        <a href="#" class="le-logo"><img src="./images/lemorsels-logo.PNG" alt="#" class="le-logo2"></a> 
+
+         <div class="desktop-nav-lnks">
+            <ul class="menu-ul-lnks">
+                <li><a href="./abouts.html" class="menu-links">About</a></li>
+                <li><a href="#" class="menu-links">Program</a></li>
+                <li><a href="#" class="menu-links">Join</a></li>
+                <li><a href="#" class="menu-links">Sponsor</a></li>
+                <li><a href="#" class="menu-links">News</a></li>
+            </ul>
+            <button class="menu-button" type="button">AS Campaign</button>
+          </div>
+        </div>
+        
+      </nav>
+  </header>
+      
+         <section class="headline-container">
+
+            <div class="Le-title">
+               <p class="catch-phrase">"Hello, organic food lovers!"</p>
+               <h1 class="Le-Slogan">LeMorsels, IT'S NATURAL AND ORGANIC AS IT SHOULD BE!</h1>
+            </div>
+          <div class="below-slogan">
+            <p class="Lemorse-quote">
+            Healthy eating is a way of life, so it's important to establish routines that are simple, realistically, and ultimately livable. 
+            LeMorsels is working with over 80 countries to help their communites to gain a better undertanding of a healthy meal life.</p>
+            <div class="small-info">
+              <p class="lemorse-date">2022.09.12(MON) ~ 16(FRI)</p>
+              <p class="le-located">@Caracas, Venezuela, LeMorsels Organic research lab.</p>
+            </div> 
+          </div>
+        </section>
+       </div>
+
+       <section class="main-program">
+ 
+        <div class="main-center"> 
+         <h2 class="main-t">Main information</h2>
+         <div class="underline"></div>
+         
+         <div class="cards-vessel">
+           <div class="card">
+               <img src="images/join-icon.png" alt="Icons8" class="icon-size">
+               <p class="card-t">Reach Out</p>
+               <p class="card-context">If you are interested in joining our comunity you can follow us on facebook or you are food scientist don't hesitate to contact us.</p>
+           </div>
+           <div class="card">
+               <img src="images/insight-icon.png" alt="Icons8" class="icon-size">
+               <p class="card-t"> Insigth</p>
+               <p class="card-context">Learn more how the sociaty keeps eating over the world is affecting their health, making their bodies more poisonous and is ruining our everyday life.</p>
+           </div>
+           <div class="card">
+               <img src="images/forum-icon.png" alt="Icons8" class="icon-size">
```

Comment:
- Kindly fill the `alt` attribute with values for those without visual accessibility. It's a project requirement.